encodings/fastlanes/src/bitpacking/unpack_iter.rs

```diff
@@ -0,0 +1,273 @@
+use std::mem;
+use std::mem::MaybeUninit;
+
+use fastlanes::BitPacking;
+use lending_iterator::gat;
+use lending_iterator::prelude::Item;
+#[gat(Item)]
+use lending_iterator::prelude::LendingIterator;
+use vortex_array::Array;
+use vortex_array::builders::UninitRange;
+use vortex_buffer::ByteBuffer;
+use vortex_dtype::NativePType;
+
+use crate::BitPackedArray;
+
+const CHUNK_SIZE: usize = 1024;
```

Comment:
we should probably make this `pub` since it's exposed as part of the return type

Reply:
actually maybe not. it's only used as part of a type bound